source/loader/ur_ldrddi.cpp

```diff
@@ -53,6 +53,9 @@ __urdlllocal ur_result_t UR_APICALL urAdapterGet(
     size_t adapterIndex = 0;
     if (nullptr != phAdapters && NumEntries != 0) {
         for (auto &platform : context->platforms) {
+            if (platform.initStatus != UR_RESULT_SUCCESS) {
```

Comment:
this should be changed in the template instead: https://github.com/oneapi-src/unified-runtime/blob/main/scripts/templates/ldrddi.cpp.mako#L59

Reply:
Thanks, done!